usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
+1) Gluster ones in ```gluster``` directory and
+2) Ceph ones in ```ceph``` directory.
```

Comment:
Please fix the syntax there, to _proper_ numbered list (each item on new line, this one generated shows all in one line).